hal_lucidlabs_helios2:
  ros__parameters:
    output_topic: "/point_cloud"
    threshold: 200
    spatial_filter: true
    flying_filter: true
    accumulate_frames: 1
    frame_id: "camera"
    # Options dumped with arv-tool
      #Enumeration: 'ExposureTimeSelector'
        #EnumEntry: (0) 'Exp62_5Us'
        #EnumEntry: (1) 'Exp250Us'
        #EnumEntry: (2) 'Exp1000Us'
    exposure_level: 0
    # Options dumped with arv-tool
      #Enumeration: 'Scan3dOperatingMode'
        #EnumEntry: (0) 'HighSpeedDistance2500mmSingleFreq'
        #EnumEntry: (1) 'HighSpeedDistance1250mmSingleFreq'
        #EnumEntry: (2) 'HighSpeedDistance625mmSingleFreq'
        #EnumEntry: (3) 'Distance8300mmMultiFreq'
        #EnumEntry: (4) 'Distance6000mmSingleFreq'
        #EnumEntry: (5) 'Distance5000mmMultiFreq'
        #EnumEntry: (6) 'Distance4000mmSingleFreq'
        #EnumEntry: (7) 'Distance3000mmSingleFreq'
        #EnumEntry: (8) 'Distance1250mmSingleFreq'
    mode: 5